traits/stubs_tests/examples/Instance.py

```diff
@@ -8,14 +8,16 @@
 #
 # Thanks for using Enthought open source!
 
-from traits.api import HasTraits, Instance, Str
+import typing
+
+from traits.api import HasTraits, Instance
 
 
 class Fruit:
-    info = Str("good for you")
+    info: str
```

Comment:
Drive-by change: it looks as though `Fruit` was trying to pretend to be a `HasTraits` class but without actually inheriting from `HasTraits`, so the `__init__` implementation didn't make a lot of sense. I've changed it to be a plain old Python class.